Dockerfile

```diff
@@ -1,22 +1,43 @@
-FROM golang:1.10 AS build
+FROM registry.access.redhat.com/ubi8/ubi-minimal AS build
 
-ARG VERSION="undefined"
-ENV WORKDIR=/go/src/github.com/3scale/3scale-istio-adapter
+ENV GOPATH=/go
 
-ADD . ${WORKDIR}
-WORKDIR ${WORKDIR}
+RUN microdnf update --nodocs -y \
+ && microdnf install --nodocs -y findutils git go-toolset make perl-Digest-SHA \
+ && microdnf clean all -y \
+ && rm -rf /var/cache/yum
 
-RUN go get -u github.com/golang/dep/cmd/dep && \
-    make build-adapter VERSION=${VERSION} && \
-    make build-cli VERSION=${VERSION}
+ARG DEP_VERSION=v0.5.3
 
-FROM centos
+RUN mkdir -p "${GOPATH}/src/github.com/golang" \
+ && cd "${GOPATH}/src/github.com/golang" \
+ && git clone --depth 1 --recurse-submodules --shallow-submodules \
+      --branch "${DEP_VERSION}" https://github.com/golang/dep.git dep \
+ && cd dep \
+ && mkdir -p ${GOPATH}/bin \
```

Comment:
👀 not consistent quoting :) the cd is quoted

Reply:
good catch :+1:

Reply:
resolved